src/helm/benchmark/static/schema_classic.yaml

```diff
@@ -534,6 +534,27 @@ metrics:
     description: The accuracy for the 10% of predictions that the model is most confident on (only computed for classification tasks).
     lower_is_better: false
 
+  # Vision Language metrics:
```

Comment:
Do we need to add the VLM metrics to the classic schema file?

Reply:
Probably not, let me revert it